docs/install.rst

```diff
@@ -437,46 +437,59 @@ Building a python interface to a locally compiled library
 -----------------------------------------------------------------------
 
 Recall that the basic user may simply ``pip install finufft``,
-then check it worked via::
+then check it worked via either (if you have ``pytest`` installed)::
 
-  python3 python/test/run_accuracy_tests.py
+  pytest python/finufft/test
+
+or the older-style eyeball check with::
+
+  python3 python/finufft/test/run_accuracy_tests.py
+
+which should report errors around ``1e-6`` and throughputs around 1-10 million points/sec.
 
 However, a user or developer may want to build a python wrapper to their locally
 compiled FINUFFT library, perhaps for more speed. We now describe this,
 for all OSes.
-We assume python3 (hence pip3).
-First make sure you have pip
-installed, and that you can already compile the C++ library (eg via ``make test``).
-Next make sure you have the required python packages::
+We assume ``python3`` (hence ``pip3``; make sure you have that installed).
+
+First, compile the shared C++ library, via, eg ``make lib -j`` (using the old-style ``makefile``),
+or via::
+
+  make -p build
+  cd build
+  cmake ..
+  cmake --build . -j
+  cp -f libfinufft.so ../lib/
```

Comment:
Joakim, could you remove this if you're now sure the py install works without it - thanks! A